src/test/java/org/scalameter/examples/JBenchExample1.java

```diff
@@ -0,0 +1,145 @@
+package org.scalameter.examples;
+
+import java.util.Random;
+import org.scalameter.*;
+import org.scalameter.japi.*;
+import org.scalameter.japi.annotation.*;
+
+
+@scopes({
+    @scopeCtx(scope = "arrays", context = "root")
+})
+public class JBenchExample1 extends JBench.Quick {
+  private Random random;
+
+  public final Context root = new ContextBuilder()
+      .put("exec.benchRuns", 20)
+      .put("exec.maxWarmupRuns", 10)
+      .build();
+
+  public final Gen<int[]> arrays = JGen.range("size", 10000, 100000, 30000)
+      .map(new Fun1<Integer, int[]>() {
+        @Override
+        public int[] apply(Integer v) {
+          return new int[v];
+        }
+      })
+      .asScala();
```

Comment:
First one is the cleaner solution, but it could be annoying for the Scala users who would like to use that annotation based frontend. 
So I think that I would go for the second, if we want to save those `asScala` calls.